### PR TITLE
Provide docs on how to use the auth with graphql

### PR DESCRIPTION
**Pull request by wesbos**
One road bump I hit to trying out strapi is that no requests work out of the box with Graphql because of the authorization. And there aren't any docs showing how to do this.

### CODE REVIEW

**Review by derrickmehaffy — DISMISSED** [review]
LGTM

**Comment by MattieBelt** [comment]
@derrickmehaffy as you mentioned in #8036, this also updates the beta docs. But it's a good addition to [the stable docs](https://github.com/strapi/strapi/blob/documentation/docs/v3.x/plugins/graphql.md).

**Comment by derrickmehaffy** [comment]
> @derrickmehaffy as you mentioned in #8036, this also updates the beta docs. But it's a good addition to [the stable docs](https://github.com/strapi/strapi/blob/documentation/docs/v3.x/plugins/graphql.md).

🙈 You are correct, @wesbos same here can you change it over to the v3 docs folder please?

File: docs/3.0.0-beta.x/plugins/graphql.md [hunk 1]
```diff
@@ -36,10 +36,25 @@ strapi install graphql
...
+
+```graphql
+mutation {
+  login(input: { identifier: "email", password: "password" }) {
+    jwt
+  }
+}
+```
+
+Then on each request, send along an `Authorization` header in the form of `{ Authorization: "Bearer YOUR_JTW_GOES_HERE" }`. This can be set in the HTTP Headers section of your GraphQL Playground.
```
Comment (amitav13) [inline docs/3.0.0-beta.x/plugins/graphql.md:51]: Tiny typo - JWT*

**Comment by wesbos** [comment]
Sorry! Pr'd it and fixed the spelling here: https://github.com/strapi/strapi/pull/8154

**Pull request closed without merging**